bin/win/publish.cmd

```diff
@@ -8,4 +8,5 @@ if exist %~dp0\dist (
     rmdir /S /Q %~dp0\dist
 )
 
-CALL %~dp0\dotnet.cmd publish %~dp0\..\..\src\AzureAuth\AzureAuth.csproj --self-contained true -r win10-x64 -c release -o %~dp0\..\..\dist %*
+
+CALL %~dp0\dotnet.cmd publish %~dp0\..\..\src\AzureAuth\AzureAuth.csproj --self-contained true -r win-x64 -c release -o %~dp0\..\..\dist %*
```

Comment:
It's unfortunate the way GitHub is rendering this diff. I think this is just swapping out `-r win10-x64` for `-r win-x64`, but there's been a newline added before this call and now there's no newline after the call. @keyuxuan, can you adjust the newlines to be the same as they were before?

Reply:
sure, fixed!